moodle/ruleset.xml

```diff
@@ -13,6 +13,16 @@
         <type>warning</type>
```

Comment:
If the rule type=error is being removed from moodle-extra to resolve #58, then this should be an error in moodle.

Otherwise, it probably shouldn't be removed from moodle-extra.